### DISCUSSION TITLE
Feature parameters file for the BraTS-TCGA-LGG radiomic features table

### DISCUSSION THREAD

**Discussion by piiq** [CBICA/CaPTk#1439] [Q&A]
Hi 👋
Can you kindly share the .csv parameters file that was used to generate the features in the Segmentation Labels and [Radiomic Features for the Pre-operative Scans of the TCGA-LGG collection (BraTS-TCGA-LGG)](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=24282668)?

**Comment by fedorov** [comment]
@sarthakpati @sbakas would this be possible? 

Ultimately, we would like these radiomics features (or a new set of features re-extracted on the original images using CapTk or pyradiomics) available within [NCI Imaging Data Commons](https://imaging.datacommons.cancer.gov/), but we need to understand what they are to at least try to harmonize the nomenclature.

On a related note, do you have mapping between the features extracted by CapTk and IBSI nomenclature?

**Reply by sarthakpati** [reply]
Hi @fedorov,

We are currently looking into this and will keep you updated.

> On a related note, do you have mapping between the features extracted by CapTk and IBSI nomenclature?

You mean the nomenclature? From my recollection, they are the same.

**Reply by fedorov** [reply]
> We are currently looking into this and will keep you updated.

Great, thanks a lot!

>> mapping between the features extracted by CapTk and IBSI nomenclature

> From my recollection, they are the same.

Sounds promising! @piiq can you take a look into this?

**Reply by piiq** [reply]
> We are currently looking into this and will keep you updated.

Hi @sarthakpati 👋! Any update on this?

I was looking into CaPTk's source code to understand the feature names and some of the things now make sense. For example:
|Suffix|Description|
|:---|:---|
|ET |Segmentation Labels: Enhancing tumor (Label-4)|
|NET|Segmentation Labels: Non-enhancing tumor (Label-1)|
|ED|Segmentation Labels: Edema (Label-2)|
|TC|NET+ET|
|WT|TC+ED|

The `OVER` suffix indicates overlap between 2 labels.

I couldn't yet figure out what is `GLOBAL` in columns like `TEXTURE_GLOBAL_NET_T1Gd_Kurtosis` so the settings for the feature extractor would really help. Let me know if I can do anything to help search for that data.

**Comment by fedorov** [comment]
Since there were no updates on this issue, we will work on extracting radiomics features using [pyradiomics](https://pyradiomics.readthedocs.io/en/latest/index.html), until we have further clarification about the nature of features produced by CapTk.

**Reply by sbakas** [reply]
Hi Andrey,

I am afraid that these specific features were extracted from CaPTk prior to introducing the parameter files. It seems that the best way forward would be to re-extract on the original images using CaPTk's default parameter file and make them available within IDC.

Please let us know of any nomenclature definition that might not be clear in the documentation.

**Reply by fedorov** [reply]
@sbakas thanks a lot for your response!

> the best way forward would be to re-extract on the original images using CaPTk's default parameter file

Can you point us to the default parameter file you have in mind, to avoid any ambiguities?

**Reply by sbakas** [reply]
For these input data, please consider up to row 29 in the default parameter file: https://github.com/CBICA/CaPTk/blob/master/src/applications/FeatureExtraction/data/1_params_default.csv

**Reply by fedorov** [reply]
Excellent, this is very helpful. Thank you Spyros!
